sharding/indexHashedNodesCoordinator.go

```diff
@@ -48,23 +52,24 @@ type EpochStartEventNotifier interface {
 }
 
 type indexHashedNodesCoordinator struct {
-	hasher                  hashing.Hasher
-	shuffler                NodesShuffler
+	marshalizer                   marshal.Marshalizer
+	hasher                        hashing.Hasher
+	shuffler                      NodesShuffler
 	epochStartRegistrationHandler EpochStartEventNotifier
-	bootStorer              storage.Storer
-	selfPubKey              []byte
-	nodesConfig             map[uint32]*epochNodesConfig
-	mutNodesConfig          sync.RWMutex
-	currentEpoch            uint32
-	savedStateKey           []byte
-	mutSavedStateKey        sync.RWMutex
-	numTotalEligible        uint64
-	shardConsensusGroupSize int
-	metaConsensusGroupSize  int
-	nodesPerShardSetter     NodesCoordinatorHelper
-	consensusGroupCacher    Cacher
-	shardIDAsObserver       uint32
-	loadingFromDisk         atomic.Value
+	bootStorer                    storage.Storer
+	selfPubKey                    []byte
+	nodesConfig                   map[uint32]*epochNodesConfig
+	mutNodesConfig                sync.RWMutex
+	currentEpoch                  uint32
+	savedStateKey                 []byte
+	mutSavedStateKey              sync.RWMutex
+	numTotalEligible              uint64
+	shardConsensusGroupSize       int
+	metaConsensusGroupSize        int
+	nodesPerShardSetter           NodesCoordinatorHelper
```

Comment:
could be renamed to nodesCoordinatorHelper

Reply:
done

Reply:
ok